numpy/ma/core.pyi

```diff
@@ -1291,8 +1291,11 @@ def size(obj: ArrayLike, axis: SupportsIndex | None = None) -> int: ...
 def diff(a, /, n=..., axis=..., prepend=..., append=...): ...
 def where(condition, x=..., y=...): ...
 def choose(indices, choices, out=..., mode=...): ...
+# NOTE: np.ma.round_ is deprecated. Use np.round instead.
+@deprecated("np.ma.round_ is deprecated. Please use np.round instead.")
```

Comment:
This message should match that of `numpy/ma/core.py` in #28899, which at the moment would be

```suggestion
@deprecated("`ma.round_` is deprecated and will be removed in a future release. Use `ma.round` instead.")
```